scripts/merge-conflicts.sh

```diff
@@ -0,0 +1,30 @@
+originalUrl="https://github.com/ethereum/solidity.git"
+git remote add english $originalUrl
+
+git config user.name "$bot_username"
+git config user.email "$bot_email"
+
+## Fetch from translated origin
+git checkout
+git fetch english --quiet
+hash=$(git rev-parse  --short=10 english/develop)
+
+syncBranch="sync-$hash"
+
+# pass the hash and the branch name to action "create PR"
+echo ::set-output name=hash::"$hash"
+echo ::set-output name=branchName::"$syncBranch"
```

Comment:
This is very CI specific, could we keep it in the workflow definition?

Reply:
I think it would be quite hard to achieve, I wanted to pass to GitHub action the branch name generated by the shell script.

Reply:
I do not think that passing variables to GitHub Action via file is possible. I found those ways:
- repository variable
- env variable
- `echo ::set-output`

Reply:
I meant that you can still do the `::set-output` thing but keep it in YAML and just set it to a value read from file. Using this weird GH mechanism is really not a great way to return values IMO. I'd keep it confined to the yaml file and stick to normal shell syntax inside scripts themselves.